book/07-git-tools/sections/rewriting-history.asc

```diff
@@ -172,7 +172,7 @@ Each time, Git will stop, let you amend the commit, and continue when you're fin
 ==== Reordering Commits
 
 You can also use interactive rebases to reorder or remove commits entirely.
-If you want to remove the ``Add cat-file'' commit and change the order in which the other two commits are introduced, you can change the rebase script from this:
+If you want to remove the "`added cat-file`" commit and change the order in which the other two commits are introduced, you can change the rebase script from this:
```

Comment:
```suggestion
If you want to remove the "`Add cat-file`" commit and change the order in which the other two commits are introduced, you can change the rebase script from this:
```

I think we're somehow reverting a change that was made on `master`.
The rest of the file uses `Add cat-file` as well.